### PR TITLE
releaseExcessTokens, make require msgs descriptive

### PR DESCRIPTION
**Pull request by arminvoid**

### CODE REVIEW

File: contracts/MaticTokenVesting.sol
Comment (jdkanani) [inline contracts/MaticTokenVesting.sol]: Let's ask auditors for best practices for error messages.

File: contracts/MaticTokenVesting.sol
Comment (jdkanani) [inline contracts/MaticTokenVesting.sol:16]: We are not using `SafeMath` in code or I am missing something?

File: contracts/MaticTokenVesting.sol
Comment (jdkanani) [inline contracts/MaticTokenVesting.sol]: No SafeMath?

File: contracts/MaticTokenVesting.sol
Comment (jdkanani) [inline contracts/MaticTokenVesting.sol]: I think this will create `deadlock`. We are adding most vesting periods in constructor and when we deploy this contract, we may not have no tokens in contract.

File: contracts/MaticTokenVesting.sol
Comment (jdkanani) [inline contracts/MaticTokenVesting.sol]: SafeMath?

**Review by jdkanani — APPROVED** [review]

**Review by jdkanani — CHANGES_REQUESTED** [review]
Approved by mistake.

File: contracts/MaticTokenVesting.sol
Comment (arminvoid) [reply contracts/MaticTokenVesting.sol:16]: Oops, didn't safeMathify the ops. My bad, sorry :(

File: contracts/MaticTokenVesting.sol
Comment (arminvoid) [reply contracts/MaticTokenVesting.sol]: Got it. Didn't know test data in the constructor will be replaced by actual vestings. I thought it's just there to test something and will be removed.

File: contracts/MaticTokenVesting.sol
Comment (arminvoid) [reply contracts/MaticTokenVesting.sol]: @vaibhavchellani suggested fission msgs. That looks nice. 
Can we establish a direct line with auditors?

**Review by jdkanani — APPROVED** [review]

**Comment by jdkanani** [comment]
@atvanguard Is `package-lock.json` updated as per new `package.json`? If not, just run `npm install` and push new `package-lock.json`

**Comment by 0xAshish** [comment]
@atvanguard will wait for 5-10 minutes here once we merge my PR #8 into this one, just in case something left.

**Pull request merged**
